README.md

```diff
@@ -282,43 +286,50 @@ coverage report
 ```yml
 language: python
 python:
+  - "3.7"
+  - "3.6"
   - "3.5"
-
+  - "3.4"
 cache:
   directories:
     - $HOME/.pip-cache/
+env:
+  global:
+    - PIPENV_VENV_IN_PROJECT=1
+    - PIPENV_IGNORE_VIRTUALENVS=1
+    - LANG=en_US.UTF-8
+    - LC_ALL=en_US.UTF-8
+    - secure: 2c0250fb-dfb1-45c9-abaf-cd3d874326c5
 before_install:
   - gem install mdl
 notifications:
   email: never
-
 install:
   - pip install --upgrade pip
-  - python3 -m pip install -r requirements.txt
-  - pip3 install pytest-flake8
-  - pip3 install pytest-cov
-  - pip3 install autopep8
-  - pip3 install gspread oauth2client
-  - pip3 install coveralls
-
+  - pip install --upgrade pipenv
+  - pipenv install --dev
 script:
-  - pytest tests
+  - pipenv run pytest tests --cov-config pytest.cov --cov
+  - pipenv run black *.py --check
+  - pipenv run black tests --check
+  - pipenv run flake8 tests*
+  - pipenv run flake8 *.py
+  - pipenv run pylint tests
+  - pipenv run pylint *.py
   - mdl README.md
-  - coverage run --source tests -m py.test
-  - coverage report
 ```
 
-## Activating Coveralls
+## Activating CodeCov
```

Comment:
Again, this is not relevant in the README. Perhaps some additional documentation elsewhere on how to use codecov or travis would be useful, but it does not belong in the readme, since everyone who uses this tool does not need to do this.

Reply:
I have deleted this section from the README.